components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m

```diff
@@ -138,6 +138,9 @@ - (MDCActionSheetAction *)action {
 }
 
 - (void)setActionFont:(UIFont *)actionFont {
+  if ([actionFont isEqual:_actionFont]) {
```

Comment:
This looks like it might break support for `MDCFontLoader` fonts since it doesn't check if `mdc_scalingCurve` is equivalent (or set). 

Is there a way to avoid reassigning fonts when the trait collection changes that doesn't require this check in `setActionFont:`?